src/ai/backend/storage/cephfs/__init__.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+from typing import Dict, List
+from uuid import UUID
+
+from ai.backend.common.types import BinarySize
+
+from ..exception import ExecutionError
+from ..types import FSUsage, VFolderCreationOptions
+from ..vfs import BaseVolume
+
+
+async def read_file(loop: asyncio.AbstractEventLoop, filename: str) -> str:
+    def _read():
+        with open(filename, "r") as fr:
+            return fr.read()
+
+    return await loop.run_in_executor(None, lambda: _read())
```

Comment:
You can just throw `_read` itself as a function, without wrapping it with lambda. 
```suggestion
    return await loop.run_in_executor(None, _read)
```